team_test.go

```diff
@@ -132,13 +129,69 @@ func TestTeamsRead(t *testing.T) {
 		assert.Equal(t, err, ErrResourceNotFound)
 	})
 
-	t.Run("without a team ID", func(t *testing.T) {
+	t.Run("without a valid team ID", func(t *testing.T) {
 		tm, err := client.Teams.Read(ctx, badIdentifier)
 		assert.Nil(t, tm)
 		assert.EqualError(t, err, "invalid value for team ID")
 	})
 }
 
+func TestTeamsUpdate(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	tmTest, tmTestCleanup := createTeam(t, client, orgTest)
+	defer tmTestCleanup()
+
+	t.Run("with valid options", func(t *testing.T) {
+		options := TeamUpdateOptions{
+			Name: String("foo bar"),
+			OrganizationAccess: &OrganizationAccessOptions{
+				ManagePolicies:    Bool(false),
+				ManageVCSSettings: Bool(true)},
+		}
+
+		tm, err := client.Teams.Update(ctx, tmTest.ID, options)
+		require.NoError(t, err)
+
+		refreshed, err := client.Teams.Read(ctx, tmTest.ID)
+		require.NoError(t, err)
+
+		for _, item := range []*Team{
+			tm,
+			refreshed,
+		} {
+			assert.Equal(t, *options.Name, item.Name)
+			assert.Equal(t,
+				options.OrganizationAccess.ManagePolicies,
+				Bool(item.OrganizationAccess.ManagePolicies),
```

Comment:
These are better written as:

```
	*options.OrganizationAccess.ManagePolicies,
	item.OrganizationAccess.ManagePolicies,
```

So dereferencing the existing pointer instead of creating a new one (which is the dereferenced in the assert function).